test/premium-schedules/premium-schedules.api-test.ts

```diff
@@ -0,0 +1,174 @@
+import { INestApplication } from '@nestjs/common';
+import Chance from 'chance';
+
+import { Api } from '../api';
+import { CreateApp } from '../createApp';
+
+const chance = new Chance();
+
+describe('Premium schedules', () => {
+  let app: INestApplication;
+  let api;
```

Comment:
we can add the type to this as well `let api: Api;`

Reply:
Changed.